std-bits/table/src/main/java/org/enso/table/data/column/operation/map/text/StringStringLongOp.java

```diff
@@ -0,0 +1,75 @@
+package org.enso.table.data.column.operation.map.text;
+
+import org.enso.table.data.column.builder.StringBuilder;
+import org.enso.table.data.column.operation.map.BinaryMapOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemAggregator;
+import org.enso.table.data.column.storage.SpecializedStorage;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.data.column.storage.StringStorage;
+import org.enso.table.data.column.storage.numeric.LongStorage;
+import org.enso.table.data.column.storage.type.TextType;
+import org.enso.table.error.UnexpectedTypeException;
+import org.graalvm.polyglot.Context;
+
+public abstract class StringStringLongOp<T>
+    extends BinaryMapOperation<String, SpecializedStorage<String>> {
+  public StringStringLongOp(String name) {
+    super(name);
+  }
+
+  protected abstract String doString(String a, Long b);
```

Comment:
```suggestion
  protected abstract String doString(String a, long b);
```

Looking at the implementation, I'm 90% sure that boxing is unnecessary here, so we might just avoid it (not sure if it will give any real difference to performance, but my rule of thumb is to not box primitives if there is no need to do so).

Reply:
```suggestion
  protected abstract String doOperation(String a, long b);
```

nitpick: I'm not sure about the `doString` name because one arg is a String but the other is not, maybe we can use something else? (Also happy to keep as-is, it's not a very important note).

Reply:
Fixed and changed name